server/src/internalClusterTest/java/org/opensearch/search/nested/SimpleNestedIT.java

```diff
@@ -455,51 +452,6 @@ public void testDeleteNestedDocsWithAlias() throws Exception {
         assertDocumentCount("test", 6);
     }
 
-    public void testExplain() throws Exception {
```

Comment:
I think there is still value in keeping this specific test with single doc and asserting for exact explain string as done here. For concurrent path, it will run with 1 slice and still we should have the same output. You can remove `indexRandomForConcurrentSearch` from the test and rename it to `testExplainWithSingleDoc`

Reply:
makes sense to add a test case for single doc as well, but for concurrent search use-case since it will run with 1 slice, the test will become somewhat redundant. So, I am thinking to add a test for single doc in the new test class introduced that has explain related test already and test only for non-concurrent use-case. Thoughts?

Reply:
Not really, it will verify that with 1 slice and concurrent search path, there is no change in explain API output. The other test class is capturing the multi-slice case.

Reply:
since we want to test for 1 slice and concurrent search path, I will add the test in SimpleNestedIT as in the new class there is no parameterization despite of having explain based tests.